diff --git a/libavutil/time.c b/libavutil/time.c
index afa6658aa6..f8d33b0651 100644
--- a/libavutil/time.c
+++ b/libavutil/time.c
@@ -55,7 +55,7 @@ int64_t av_gettime(void)
 
 int64_t av_gettime_relative(void)
 {
-#if HAVE_CLOCK_GETTIME && defined(CLOCK_MONOTONIC)
+#if 0 && HAVE_CLOCK_GETTIME && defined(CLOCK_MONOTONIC)
 #ifdef __APPLE__
     if (clock_gettime)
 #endif
@@ -70,7 +70,7 @@ int64_t av_gettime_relative(void)
 
 int av_gettime_relative_is_monotonic(void)
 {
-#if HAVE_CLOCK_GETTIME && defined(CLOCK_MONOTONIC)
+#if 0 && HAVE_CLOCK_GETTIME && defined(CLOCK_MONOTONIC)
 #ifdef __APPLE__
     if (!clock_gettime)
         return 0;
